doc/update024.md

```diff
@@ -0,0 +1,37 @@
+### dPoW 0.2.4 update informations:
+
+**The Marmarachain's codebase is ready for Season 4 Activation in 3P node**
+
+- Please update your Hush codebase to [03dd780](https://github.com/marmarachain/Marmara-v.1.0/tree/03dd78037067ebb27af8b33f6adcdbede3813007) and build it
```

Comment:
Are you sure about that?

Reply:
@TheComputerGenie is your comment about MCL's readiness or the mentioned commit?

Reply:
ah, I spotted the Hush mention. will correct it. please comment if you find any other mistakes